test/expected/delete-14.out

```diff
@@ -111,37 +111,38 @@ WHERE series_1 IN (SELECT series_1 FROM "two_Partitions" WHERE series_1 > series
 EXPLAIN (costs off)
 DELETE FROM "two_Partitions"
 WHERE series_1 IN (SELECT series_1 FROM "two_Partitions" WHERE series_1 > series_val());
-                                                                    QUERY PLAN                                                                    
---------------------------------------------------------------------------------------------------------------------------------------------------
- Delete on "two_Partitions"
-   Delete on "two_Partitions" "two_Partitions_2"
-   Delete on _hyper_1_1_chunk "two_Partitions_3"
-   Delete on _hyper_1_2_chunk "two_Partitions_4"
-   Delete on _hyper_1_3_chunk "two_Partitions_5"
-   Delete on _hyper_1_4_chunk "two_Partitions_6"
-   ->  Hash Join
-         Hash Cond: ("two_Partitions".series_1 = "two_Partitions_1".series_1)
-         ->  Append
-               ->  Seq Scan on "two_Partitions" "two_Partitions_2"
-               ->  Seq Scan on _hyper_1_1_chunk "two_Partitions_3"
-               ->  Seq Scan on _hyper_1_2_chunk "two_Partitions_4"
-               ->  Seq Scan on _hyper_1_3_chunk "two_Partitions_5"
-               ->  Seq Scan on _hyper_1_4_chunk "two_Partitions_6"
-         ->  Hash
-               ->  HashAggregate
-                     Group Key: "two_Partitions_1".series_1
-                     ->  Append
-                           ->  Seq Scan on "two_Partitions" "two_Partitions_7"
-                                 Filter: (series_1 > (series_val())::double precision)
-                           ->  Index Scan using "_hyper_1_1_chunk_two_Partitions_timeCustom_series_1_idx" on _hyper_1_1_chunk "two_Partitions_8"
-                                 Index Cond: (series_1 > (series_val())::double precision)
-                           ->  Index Scan using "_hyper_1_2_chunk_two_Partitions_timeCustom_series_1_idx" on _hyper_1_2_chunk "two_Partitions_9"
-                                 Index Cond: (series_1 > (series_val())::double precision)
-                           ->  Index Scan using "_hyper_1_3_chunk_two_Partitions_timeCustom_series_1_idx" on _hyper_1_3_chunk "two_Partitions_10"
-                                 Index Cond: (series_1 > (series_val())::double precision)
-                           ->  Index Scan using "_hyper_1_4_chunk_two_Partitions_timeCustom_series_1_idx" on _hyper_1_4_chunk "two_Partitions_11"
-                                 Index Cond: (series_1 > (series_val())::double precision)
-(28 rows)
+                                                                       QUERY PLAN                                                                       
```

Comment:
Do we have tests for these cases?
1. Triggers for delete (before/after row and stmt)
2. Delete with join on the table  (from PG docs below) 
DELETE FROM films USING producers
  WHERE producer_id = producers.id AND producers.name = 'foo';
3. DELETE ... with RETURNING *

Reply:
ok added test for DELETE RETURNING, there were already tests for the other stuff